atom/browser/node_debugger.cc

```diff
@@ -25,10 +27,15 @@ void NodeDebugger::Start(node::MultiIsolatePlatform* platform) {
   node::DebugOptions options;
   for (auto& arg : base::CommandLine::ForCurrentProcess()->argv()) {
 #if defined(OS_WIN)
-    options.ParseOption("Electron", base::UTF16ToUTF8(arg));
+    const std::string nice_arg = base::UTF16ToUTF8(arg);
 #else
-    options.ParseOption("Electron", arg);
+    const std::string nice_arg = arg;
```

Comment:
This second case can be made a reference to avoid an unnecessary `std::string` being created:

```cpp
const std::string& nice_arg = arg;
```